pydantic_evals/pydantic_evals/evaluators/llm_as_a_judge.py

```diff
@@ -168,22 +215,7 @@ async def judge_input_output_expected(
     If the model is not specified, a default model is used. The default model starts as 'openai:gpt-4o',
     but this can be changed using the `set_default_judge_model` function.
     """
-    user_prompt = dedent(
-        f"""
-        <Input>
-        {_stringify(inputs)}
-        </Input>
-        <ExpectedOutput>
-        {_stringify(expected_output)}
-        </ExpectedOutput>
-        <Output>
-        {_stringify(output)}
-        </Output>
-        <Rubric>
-        {rubric}
-        </Rubric>
-        """
-    )
+    user_prompt = _build_prompt_with_input(inputs, output, rubric, expected_output)
```

Comment:
Can we rename this function `_build_prompt` and also use it in the cases where there's no `inputs`? We can make `inputs` and `expected_output` arguments both optional and only add them in if they're provided